source/blog/sitemap.xml.erb

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9">
-  <% data&.blog&.posts&.map {|p| p[1] }.select{ |p| p[:created_at] && p[:title] }.sort_by { |p| p[:created_at] }.reverse.take(2).each do |post| %>
-    <% if !post.title.blank? %>
-      <url>
-        <loc><%= post_url(post) %></loc>
-        <news:news>
-          <news:publication>
-            <news:name>Sharesight</news:name>
-            <news:language>en</news:language>
-          </news:publication>
-          <news:genres>Blog</news:genres>
-          <news:publication_date><%= Date.parse(post.created_at.to_s) %></news:publication_date>
-          <news:title><%=post.title.gsub('&','&amp;')%></news:title>
-        </news:news>
-      </url>
-    <%end%>
-  <%end%>
+  <% blog_posts(order: :latest_first).select{ |p| p[:title] }.take(2).each do |post| %>
```

Comment:
Can `p[:title]` be an empty string? Previously we checked for `blank?`, which covered both. But the code itself is clearer now. 👍

Reply:
👍 Just copied 1:1 what we are using in the main sitemap actually—`post[:title]&.present?` (plus the `.take(2)`) in ac26bc28